### PR TITLE
WORK IN PROGRESS Cleanup

### PR DESCRIPTION
**Pull request by nurikk**
Use ts-test instead of babel
before
```
added 999 packages, and audited 999 packages in 18s
```
after
```
added 707 packages, and audited 707 packages in 23s
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,20 +70,10 @@
...
     "test": "jest test",
-    "test-with-coverage": "jest test --coverage --collectCoverageFrom 'src/**/*.ts'",
+    "test-with-coverage": "jest test --coverage",
     "test-watch": "jest test --watch",
     "eslint": "node_modules/.bin/eslint . --ext .ts",
     "docs": "typedoc --tsconfig typedoc-tsconfig.json"
   },
-  "version": "0.13.33",
-  "jest": {
-    "coverageThreshold": {
```
Comment (Koenkk) [inline package.json:86]: Why was this removed?

Comment (nurikk) [reply package.json:86]: It's moved to corresponding.json file

Comment (nurikk) [reply package.json:86]: jest.config.js

File: src/controller/controller.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ class Controller extends events.EventEmitter {
      *
      * To auto detect the port provide `null` for `options.serialPort.path`
      */
-    public constructor(options: Options) {
+    public constructor(options: Partial<Options>) {
```
Comment (Koenkk) [inline src/controller/controller.ts:76]: Why put partial here?

Comment (nurikk) [reply src/controller/controller.ts]: My bad, fixed
